app/api/repositories/organizer_repository.py

```diff
@@ -0,0 +1,64 @@
+from typing import Annotated
+
+from beanie import PydanticObjectId, UpdateResponse
+from beanie.operators import Set
+from fastapi import Depends
+from motor.motor_asyncio import AsyncIOMotorClientSession
+from pymongo.results import UpdateResult
+
+from app.api.errors.organizer_not_found import OrganizerNotFound
+from app.api.models.organizer_model import (
+    Organizer,
+    OrganizerDocument,
+    PartialOrganizer,
+)
+
+
+class OrganizerRepository:
+    async def find_all(self):
```

Comment:
return type

Reply:
Solved by [commit](https://github.com/liel-almog/gala/pull/9/commits/05acf752bda238c593642d5ce38a8730541859b7)

Reply:
Do I need to annotate the route handlers too?